ce/api/util.py

```diff
@@ -95,14 +96,30 @@ def mean_datetime(datetimes):
     mean = np.mean(timestamps)
     return datetime.fromtimestamp(mean, tz=timezone.utc)
 
+def validate_cell_method(cell_method):
+    return cell_method in ('mean', 'standard_deviation')
+
 def search_for_unique_ids(sesh, ensemble_name='ce', model='', emission='',
-                          variable='', time=0, timescale=''):
+                          variable='', time=0, timescale='', cell_method='mean'):
+    if not validate_cell_method(cell_method):
+        raise Exception('Unsupported cell_method: {}'.format(cell_method))
+
+    cell_methods = sesh.query(mm.DataFileVariable.variable_cell_methods)\
+                    .distinct(mm.DataFileVariable.variable_cell_methods).all()
+    pattern = {
+        'standard_deviation': r'time:[a-z\s]*time:\s+standard_deviation\s+over\s+days',
+        'mean': r'time:[a-z\s]*\s+(time:\s+mean\s+over\s+days)?'
```

Comment:
As just discussed in meeting with @nikola-rados, @rod-glover, and @corviday , `generate_climos` [is going to start putting out some climatologies with cell_methods that end "mean over years"](https://github.com/pacificclimate/climate-explorer-data-prep/issues/64) and this regex should support "years" in addition to "days" here.

Reply:
I'll add it in!